src/Command/CreateControllerCommand.php

```diff
@@ -66,12 +71,19 @@ private function buildController($name) {
      * @param $output
      */
     private function checkIfFileExists($name, $output) {
-        $files = new \FilesystemIterator('app/controllers');
+        $files = new \FilesystemIterator('src/Controller');
         foreach($files as $file) {
             if ($file->getFilename() == $name) {
```

Comment:
Use strict comparison in most cases you need to compare values. Again, it's about stability, decreases chances of unexpected bugs.
```suggestion
            if ($file->getFilename() === $name) {
```

Reply:
@vladimino after your comment i just read this article
https://www.copterlabs.com/strict-vs-loose-comparisons-in-php/
I never thought loose comparison would cause any problems, now i wonder why i would use it anyway !

Reply:
Great article, thanks for sharing! And you just became a little bit better PHP developer, very happy to see this 💙